providers/gcp/resources/discovery/gcr/gcr.go

```diff
@@ -52,21 +49,22 @@ func (a *GcrImages) ListRepository(repository string, recursive bool) ([]*asset.
 			repoURL := repo.String()
 			imageUrl := repoURL + "@" + digest
 
-			asset := &asset.Asset{
+			asset := &inventory.Asset{
 				PlatformIds: []string{MondooContainerImageID(digest)},
 				Name:        containerid.ShortContainerImageID(digest),
```

Comment:
this looks like the k8s container-image discovery. I think we should match the approach and just return an asset with a connection config and nothing else. We should let the os provider fill in the platform IDs, name, etc.